exemplar/src/main/java/com/squareup/exemplar/ExemplarJavaApp.java

```diff
@@ -5,16 +5,14 @@
 import misk.environment.EnvironmentModule;
 import misk.hibernate.HibernateModule;
 import misk.moshi.MoshiModule;
-import misk.web.WebActionsModule;
-import misk.web.jetty.JettyModule;
+import misk.web.jetty.WebModule;
```

Comment:
`misk.web.WebModule`?

Reply:
Perhaps maybe then it is best to also have a `misk.web.jetty.JettyModule` that is installed by `misk.web.WebModule`...

Reply:
Good call, fixed.